### ISSUE TITLE
Request for test set of SQuAD

### ISSUE THREAD

**Issue by soheeyang** [facebookresearch/DPR#62]
With `data/download_data.py`, we can download the train and dev sets of SQuAD by specifying the resource as `data.retriever.squad1-train` and `data.retriever.squad1-dev`, respectively.
I guess the QA files of SQuAD like `data.retriever.qas.trivia-test` can be made by dumping tsv files that contains only the "question" and "answers" of those files.
However, I could not find the link to the test set for SQuAD from the download script. Could you please provide the qas file for the test set of SQuAD?

Adding more details to this issue:
- I noticed that the test set size of SQuAD indicated in the DPR paper is the same as the official dev set size of SQuAD v1.1 (dev-v1.1.json), so made a file named `squad1-test.csv` by processing dev-v1.1.json. However, I could not reproduce the retrieval result reported in the paper. Therefore, I came to wonder if the test split used for the paper is a different file or processed in a different way.
- To reproduce the retriever, I used the hyperparameters specified in the README file as "Best hyperparameter settings - 2. Biencoder(Retriever) training in single set mode." referring to the comment in https://github.com/facebookresearch/DPR/issues/22. I could successfully reproduce the scores of TriviaQA, but could not get the numbers for SQuAD with the tsv file I processed by myself.

(I'm sorry for initially submitting the issue in an incomplete form; I accidentally hit the enter button while I was typing...)

**Comment by soheeyang** [comment]
In addition, could you please upload the gold_passages_info files for SQuAD and TriviaQA?

**Comment by vlad-karpukhin** [comment]
Hi Sohee,
Please download Squad's test set we used suing this url:
https://dl.fbaipublicfiles.com/dpr/data/retriever/squad1-test.qa.csv

**Comment by soheeyang** [comment]
Hi, I'm sorry for the late comment.
Thank you so much for sharing the dataset!
Somehow I still cannot reproduce the results in the paper, but will try again. Thanks!
